test/evp_kdf_test.c

```diff
@@ -174,12 +174,61 @@ static int test_kdf_ss_hash(void)
     return ret;
 }
 
+static int test_kdf_x963(void)
+{
+    int ret;
+    EVP_KDF_CTX *kctx = NULL;
+    unsigned char out[1024 / 8];
+    /*
+     * Test data from https://csrc.nist.gov/CSRC/media/Projects/
+     *  Cryptographic-Algorithm-Validation-Program/documents/components/
+     *  800-135testvectors/ansx963_2001.zip
+     */
+    const unsigned char z[] = {
```

Comment:
`static` as well?  Here and below?

Not a critical either way.

Reply:
Changed entire file to use statics